hermes/crates/hermes-ipfs/examples/provide-content-dht.rs

```diff
@@ -0,0 +1,113 @@
+//! Use Hermes IPFS to distribute content using DHT
```

Comment:
I think this should be using `dht_put` and `dht_get`? https://docs.rs/rust-ipfs/0.11.20/rust_ipfs/struct.Ipfs.html#method.dht_put

Reply:
I'll add another example with these methods, also to show how to use the underlying `ipfs` node.